typings/index.ts

```diff
@@ -483,7 +484,8 @@ assertType<Message | null>(dmChannel.lastMessage);
 assertType<Message | null>(threadChannel.lastMessage);
 assertType<Message | null>(newsChannel.lastMessage);
 assertType<Message | null>(textChannel.lastMessage);
```

Comment:
How are these...valid?

Reply:
Discord sends last_message_id for text-based channels, which those are.

Reply:
Man I am blinddddd